### ISSUE TITLE
Lighting without cubemaps appears glossy

### ISSUE THREAD

**Issue by Nolankicks** [Facepunch/sbox-issues#6527]
### Describe the bug

Lighting without cubemaps appears glossy.

### To Reproduce

1. Create a scene just with a skybox.

### Expected behavior

The lighting should look the way it did before.

### Media/Files

![image](https://github.com/user-attachments/assets/1d6930da-e623-4a3c-b5b8-99a9232dd7ae)
![image](https://github.com/user-attachments/assets/cc770926-c519-4992-b94e-3f38d9f41e59)

**Comment by handsomematt** [comment]
@samzanemesis is this intended

**Comment by samzanemesis** [comment]
No, I think I messed BRDF on skin shader that uses two lobes when attempting to fix it in standard materials

**Comment by samzanemesis** [comment]
Nope it was just outdated, and was using the old cubemap normalization curve
